share/shwap/p2p/discovery/discovery.go

```diff
@@ -34,6 +34,10 @@ const (
 	// logInterval defines the time interval at which a warning message will be logged
 	// if the desired number of nodes is not detected.
 	logInterval = 5 * time.Minute
+
+	// discovery version is a prefix for all tags used in discovery. It is bumped when
+	// there are protocol breaking changes.
+	version = "v1"
```

Comment:
The breaking changes we have are not in discovery. The breaking changes are with shrex protocol or the protocol that uses the discovery tool. So instead of doing global bump for the tool, only ensuring tag given to discovery should be enough

Reply:
The change allows to decouple versioning from tags, which is important to keep other tag relateed logic, like metric tags. We dont want metric names to be bumped with every version